src/base/bittorrent/private/speedmonitor.cpp

```diff
@@ -27,16 +27,24 @@
  * exception statement from your version.
  */
 
-#include <QList>
 #include "speedmonitor.h"
 
 void SpeedMonitor::addSample(const SpeedSample &sample)
 {
-    m_speedSamples.push_back(sample);
+    static_assert(MAX_SAMPLES > 0, "MAX_SAMPLES should > 0");
```

Comment:
`static_assert` is a declaration, not an expression, i.e. you can put it near the declaration of the constant.